src/Stratis.Bitcoin.Features.Interop/ETHClient/ETHClient.cs

```diff
@@ -188,13 +220,13 @@ public async Task<BigInteger> GetBalanceAsync(string address)
         /// <inheritdoc />
         public async Task<MultisigTransactionIdentifiers> SubmitTransactionAsync(string destination, BigInteger value, string data, int gasPrice)
         {
-            return await MultisigWallet.SubmitTransactionAsync(this.web3, this.settings.MultisigWalletAddress, destination, value, data, this.settings.GasLimit, this.settings.GasPrice).ConfigureAwait(false);
+            return await MultisigWallet.SubmitTransactionAsync(this.web3, this.settings.MultisigWalletAddress, destination, value, data, this.settings.GasLimit, gasPrice).ConfigureAwait(false);
```

Comment:
I think this may have been a merge glitch, we should always use the gasPrice parameter (the caller is responsible for fallbacks to the config)